src/style/themes.ts

```diff
@@ -49,7 +49,7 @@ export const lightTheme: DefaultTheme = {
     valid: '#4ebf08',
     highlightGradient: 'linear-gradient(45deg, rgba(18,0,218,1) 0%, rgba(255,93,81,1) 100%)'
   },
-  badge: {
+  txInfo: {
```

Comment:
This is too specific for a theme IMO. I would keep "badge".

I think we could follow [this](https://github.com/alephium/explorer/pull/55/commits/a4257809037832a555cfc8d6bca4892877017d97) approach, as we discussed earlier: "badges" are flat and can have a border. On the explorer, so far I'm using badges for certain amounts I want to highlight + tx direction (from / to). The same could be done here.

Reply:
(As you can see in my commit, I'm no longer using specific colors for badges, so this "txInfo" part of the theme could even be removed entirely... for now :) In the future, if we want to have more contrasted options, we will make sure to use secondary colors defined in the "global" property - like alert, warning, valid etc.)

Reply:
I'm a little confused 😅  

So, [currently in the wallet](https://github.com/alephium/alephium-wallet/blob/derive-new-addresses/src/components/Badge.tsx#L37-L47), the `Badge`'s text color is set from the props of the component, its background color is a more transparent version of the text color, and the border (if set) is the same as the text color. That was done because [we agreed](https://github.com/alephium/alephium-wallet/pull/131#discussion_r813888470) that "labels" are essentially "badges".

If I remove the `txInfo` from the theme, then we need a way to define the color palette for the `TransactionalInfo` component (used to display the words "↓ Received", "↑ Sent", "Pending", as well as the amounts in the transaction list tables). Those elements are not badges, since they have no background at all. Right?

Do you suggest that I move those color definitions from the theme into the `TransactionalInfo` component?

Thanks for clarifying and helping me understand!

Reply:
I need the `txInfo` colors in 2 different places, so moving them in the `TransactionalInfo` component doesn't make sense.

| Transaction lists | Transaction details |
| ---------------- | ------------------- |
| ![image](https://user-images.githubusercontent.com/1579899/156148232-df428404-a420-4dfd-adb1-6c996f257f2d.png) | ![image](https://user-images.githubusercontent.com/1579899/156148572-c2234013-4229-4364-a533-98e23948c8f9.png) |

Reply:
Ah, I'm sorry, didn't realise that this was specifically used for the `TransactionalInfo` component now. 
My comment still stands though, even more in that case - we don't want one entry in the theme that is here only to be used by one single component. 

The `TransactionalInfo` component should use colors options we already have.
In that specific case, `received` could use the `global.valid` color, `send` the `font.primary`, `pending` would be `font.secondary` etc.

Does that make sense? Sorry for the confusion.

Let's do that: I'll sync the palette with the explorer's one when doing the overall testing and cleaning, so that you don't have to spend time on this. Would that be ok for you? 

Thanks!

Reply:
Yes, that makes sense! I'll reuse existing colors, delete the `txInfo` ones, and you can update the colors when you want to!

Reply:
That's it 💯 . Thanks!